nuxt.config.ts

```diff
@@ -0,0 +1,28 @@
+import type { NuxtConfig } from '@nuxt/types'
+
+const config: NuxtConfig = {
+  build: {},
+  buildModules: [
+    '@nuxtjs/composition-api',
+    '@nuxt/typescript-build'
+  ],
+  css: [],
+  env: {},
+  head: {
+    title: 'nuxt-community/typescript-template',
+    meta: [
+      { charset: 'utf-8' },
+      { name: 'viewport', content: 'width=device-width, initial-scale=1' },
+      { hid: 'description', name: 'description', content: 'A boilerplate to start a Nuxt+TS project quickly' }
+    ],
+    link: []
+  },
+  loading: { color: '#0c64c1' },
+  modules: [],
+  plugins: [
+    '~/plugins/composition-api',
```

Comment:
We shouldn't need this as the module registers the plugin for us.

Reply:
Ah good point, I totally missed getting rid of this line which I added `@vue/composition-api` originally.